cms/envs/aws.py

```diff
@@ -35,6 +35,7 @@
 # prefix.
 CONFIG_PREFIX = SERVICE_VARIANT + "." if SERVICE_VARIANT else ""
 
+FEATURES['ENABLE_SPECIAL_EXAMS'] = True
```

Comment:
FYI - For future reference, consider putting these temporary environment variable changes in your personal `private.py` file within `cms/envs/` and `lms/envs/`.  Since `private.py` is listed in `.gitignore`, you wouldn't need a NOMERGE commit.